### PR TITLE
feat: enhance learning center to match the mockup design

### LINKED ISSUE #6221: make learning center matching the mockup design
### Is your enhancement related to a problem? Please describe

The mockup design was:
image

but current implementation is:
image

### Describe the solution you'd like

- Add the missing collapse (chevron)
- make it matching background (the title should be inside the card)
- make the text smaller

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by feloy**
### What does this PR do?

- Add the missing collapse (chevron)
- make it matching background (the title should be inside the card)
- make the text smaller

### Screenshot / video of UI

https://github.com/containers/podman-desktop/assets/9973512/28285e0d-6be3-456e-ae45-3d0e3532ec74

### What issues does this PR fix or reference?

Fixes #6221 

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

**Comment by benoitf** [comment]
great job

I think the chevron in 'non-expanded' mode is not the expected one

expanded = chevron-down 
non-expanded = chevron-right (and not chevron-up)

I would not introduce expandable UI widget as part of the UI change as it may not be the name we would or if it's really the behavior, etc.
We could say it's a Card etc

**Comment by feloy** [comment]
> great job
> 
> I think the chevron in 'non-expanded' mode is not the expected one
> 
> expanded = chevron-down non-expanded = chevron-right (and not chevron-up)
> 

I'll change

> I would not introduce expandable UI widget as part of the UI change as it may not be the name we would or if it's really the behavior, etc. We could say it's a Card etc

You mean to integrate the Expandable logic directly into LearningCenter?

File: packages/renderer/src/lib/ui/animations.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { cubicOut } from 'svelte/easing';
```
Comment (jeffmaury) [inline packages/renderer/src/lib/ui/animations.ts:1]: Missing header

File: packages/renderer/src/lib/Expandable.svelte [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<script lang="ts">
```
Comment (jeffmaury) [inline packages/renderer/src/lib/Expandable.svelte]: Wondering if this shouldn't be in the UI lib

Comment (jeffmaury) [inline packages/renderer/src/lib/Expandable.svelte]: Missing unit test on the expanded state

**Comment by slemeur** [comment]
Looking great.
Could we add a hover state on the arrows to move left/right?

**Comment by benoitf** [comment]
> You mean to integrate the Expandable logic directly into LearningCenter?

yes

File: packages/renderer/src/lib/ui/animations.ts (hunk 1 above)
Comment (benoitf) [reply packages/renderer/src/lib/ui/animations.ts:1]: FYI this move should be a small PR to move out the animations
so as it's a pure move, small change, small review, fast merge

File: packages/renderer/src/lib/ui/animations.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+export function fadeSlide(
```
Comment (benoitf) [inline packages/renderer/src/lib/ui/animations.ts:3]: moving this to a separate file will require specific test for the function as well

**Review by jeffmaury — COMMENTED** [review]
Why the new component has been removed ? That was a good idea. What's the rationale ?

**Comment by benoitf** [comment]
@jeffmaury was it a good name, could it been applied to other parts 
Should be part of a Card ? etc

**Review by benoitf — APPROVED** [review]

**Comment by feloy** [comment]
> Why the new component has been removed ? That was a good idea. What's the rationale ?

@jeffmaury -> https://github.com/containers/podman-desktop/pull/6281#issuecomment-1980367020

**Comment by feloy** [comment]
> Looking great. Could we add a hover state on the arrows to move left/right?

@slemeur -> https://github.com/containers/podman-desktop/pull/6284

**Comment by slemeur** [comment]
thanks @feloy

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**